src/layout/layout.module.css

```diff
@@ -57,6 +74,96 @@
 .header__nav a:hover {
   color: rgba(0, 0, 0, 0);
 }
+
+.container {
+  background-color: transparent;
+  border: 0;
+  font-size: 20px;
+  display: none;
+  cursor: pointer;
+  position: absolute;
+  right: 20px;
+  top: 20px;
+}
+
+.container:focus {
+  outline: none;
+  box-shadow: 0 0 0 2px rgba(21, 156, 228, 0.4);
+}
+
+.bar1,
+.bar2,
+.bar3 {
+  position: relative;
+  width: 25px;
+  height: 3px;
+  background-color: var(--color-secondary1);
+  margin: 4px 0;
+  transition: 0.4s;
+}
+
+.bar1_change,
+.bar2_change,
+.bar3_change {
+  position: relative;
+  width: 25px;
+  height: 3px;
+  background-color: var(--color-secondary1);
+  margin: 4px 0;
+  transition: 0.4s;
```

Comment:
Can use CSS-modules' `composes` to prevent duplicate code. 

`composes: bar1`

Reply:
fixed :D